### PR TITLE
Change from "in Nodejs" to "with Nodejs"

### PR DESCRIPTION
**Pull request by booleans-oss**
Node.js is a run-time environment that runs Javascript files. So it is a mistake to say that Discord.js - or a package whatsoever - is created in Node.js as it is not a language. It mainly leads to the misunderstanding of the differences between Node.js and Javascript (which also lead to people saying: "I code in Nodejs").

So I would rather say that it is written in Javascript and run with Node.js which is actually what is happening.

### CODE REVIEW

File: guide/topics/discordjs/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ What is a Discord bot? It is a "bot" account ran with code that has some special
 
 What can you do with a Discord bot? There are many uses for a Discord bot, moderating a guild, providing support (tickets, suggestions) or even playing music in voice channels and much more.
 
-What is Discord.JS? It's a library for the Discord API written in Node.JS mostly to make Discord bots, though it can be used in the browser to make a dashboard or anything else related.
+What is Discord.JS? It's a library for the Discord API written in Javascript with Node.JS mostly to make Discord bots, though it can be used in the browser to make a dashboard or anything else related.
```
Comment (acollierr17) [inline guide/topics/discordjs/README.md]: Good catch on this. However, I have a suggestion for the wording to be a bit more clear.
> What is Discord.JS? It's a Javascript library built as a Node.js module for the Discord API mainly used to make Discord bots; however, you can use it in the browser to create a dashboard for your bot and more

**Review by acollierr17 — CHANGES_REQUESTED** [review]

File: guide/topics/discordjs/README.md (hunk 1 above)
Comment (JonasSchiott) [reply guide/topics/discordjs/README.md]: Agreed, @booleans-oss feel free to make a new commit so we can merge :D

Comment (booleans-oss) [reply guide/topics/discordjs/README.md]: It's true that it seemed a bit wordy. Thanks for noting that

Comment (acollierr17) [reply guide/topics/discordjs/README.md]: No problem. Also, I forgot to mention the edit in my comment above, but there are further clarification and grammar changes. So you can check it out and use that instead. We can merge after that!

**Review by acollierr17 — APPROVED** [review]
